pkg/parca/parca.go

```diff
@@ -108,6 +120,10 @@ func Run(ctx context.Context, logger log.Logger, reg *prometheus.Registry, flags
 		return err
 	}
 
+	if flags.Mode == "scraper" {
```

Comment:
Small nit, maybe make this a const?

Reply:
Yes, I was thinking the same while implementing. Sadly we still have an occurrence in the flag kong tag that won't be possible to be a constant. I'll do that anyway. :+1: